From 3a0ed56df64a909d60334c9db20551997610221c Mon Sep 17 00:00:00 2001
From: leo barbosa <leo.barbosa@canonical.com>
Date: Mon, 29 Jan 2018 16:49:34 -0300
Subject: [PATCH] Removing test that fails for check docs/man page after
 CVE-2018-1000008 was applied

After patch CVE-2018-1000008 was applied this test starts to fail.
Since it is not related to any important feature this patch disable it
for now.

---
 tests/data/Makefile.inc |  2 +-
 tests/data/test1140     | 26 --------------------------
 2 files changed, 1 insertion(+), 27 deletions(-)
 delete mode 100644 tests/data/test1140

diff --git a/tests/data/Makefile.inc b/tests/data/Makefile.inc
index 4a37191..2445d5b 100644
--- tests/data/Makefile.inc
+++ tests/data/Makefile.inc
@@ -119,7 +119,7 @@
 test1112 test1113 test1114 test1115 test1116 test1117 test1118 test1119 \
 test1120 test1121 test1122 test1123 test1124 test1125 test1126 test1127 \
 test1128 test1129 test1130 test1131 test1132 test1133 test1134 test1135 \
-test1136 test1137 test1138 test1139 test1140 test1141 test1142 test1143 \
+test1136 test1137 test1138 test1139          test1141 test1142 test1143 \
 test1144 test1145 test1146 \
 test1152 \
 test1163 \
diff --git a/tests/data/test1140 b/tests/data/test1140
deleted file mode 100644
index b9458be..0000000
--- tests/data/test1140
+++ /dev/null
@@ -1,26 +0,0 @@
-<testcase>
-<info>
-<keywords>
-source analysis
-symbols-in-versions
-documentation
-</keywords>
-</info>
-
-#
-# Client-side
-<client>
-<server>
-none
-</server>
-
- <name>
-Verify the nroff of man pages
- </name>
-
-<command type="perl">
-%SRCDIR/nroff-scan.pl %SRCDIR/../docs/ %SRCDIR/../docs/libcurl/*.3 %SRCDIR/../docs/libcurl/opts/*.3 %SRCDIR/../docs/*.1
-</command>
-</client>
-
-</testcase>
-- 
2.7.4

